.maintain/clippy.sh

```diff
@@ -0,0 +1,120 @@
+-A clippy::len_without_is_empty \
+-A clippy::len_zero \
+-A clippy::let_and_return \
+-A clippy::many_single_char_names \
+-A clippy::map_collect_result_unit \
+-A clippy::match_like_matches_macro \
+-A clippy::match_ref_pats \
+-A clippy::mem_replace_with_default \
+-A clippy::module_inception \
+-A clippy::mut_from_ref \
+-A clippy::needless_borrow \
+-A clippy::needless_collect \
+-A clippy::needless_doctest_main \
+-A clippy::needless_range_loop \
+-A clippy::new_ret_no_self \
+-A clippy::bind_instead_of_map \
+-A clippy::comparison_to_empty \
+-A clippy::nonminimal_bool \
+-A clippy::single_match \
+-A clippy::option_as_ref_deref \
+-A clippy::map_entry \
+-A clippy::bool_comparison \
+-A clippy::explicit_counter_loop \
+-A clippy::clone_double_ref \
+-A clippy::blacklisted_name \
+-A clippy::cmp_owned \
+-A clippy::filter_map_identity \
+-A clippy::double_must_use \
+-A clippy::redundant_pattern \
+-A clippy::eval_order_dependence \
+-A clippy::from_str_radix_10 \
+-A clippy::clone_double_ref \
+-A clippy::cmp_owned \
+-A clippy::redundant_field_names \
+-A clippy::useless_format \
+-A clippy::iter_next_slice \
+-A clippy::blocks_in_if_conditions \
+-A clippy::if_let_some_result \
+-A clippy::expect_fun_call \
+-A clippy::needless_update \
+-A clippy::manual_strip \
+-A clippy::print_with_newline \
+-A clippy::manual_flatten \
+-A clippy::string_extend_chars \
+-A clippy::match_overlapping_arm \
+-A clippy::iter_nth_zero \
+-A clippy::filter_next \
+-A clippy::blocks_in_if_conditions \
+-A clippy::skip_while_next \
+-A clippy::boxed_local \
+-A clippy::manual_saturating_arithmetic \
+-A clippy::unnecessary_filter_map \
+-A clippy::write_literal \
+-A clippy::iter_cloned_collect \
+-A clippy::derive_hash_xor_eq \
+-A clippy::if_same_then_else \
+-A clippy::unit_arg \
+-A clippy::op_ref \
+-A clippy::field_reassign_with_default \
+-A clippy::needless_question_mark \
+-A clippy::map_clone \
+-A clippy::option_map_unit_fn \
+-A clippy::or_fun_call \
+-A clippy::int_plus_one \
+-A clippy::println_empty_string \
+-A clippy::ptr_arg \
+-A clippy::deref_addrof \
+-A clippy::redundant_clone \
+-A clippy::redundant_closure \
+-A clippy::redundant_pattern_matching \
+-A clippy::redundant_slicing \
+-A clippy::redundant_static_lifetimes \
+-A clippy::result_unit_err \
+-A clippy::search_is_some \
+-A clippy::should_implement_trait \
+-A clippy::single_char_add_str \
+-A clippy::single_component_path_imports \
+-A clippy::stable_sort_primitive \
+-A clippy::suspicious_else_formatting \
+-A clippy::too_many_arguments \
+-A clippy::try_err \
+-A clippy::unnecessary_lazy_evaluations \
+-A clippy::unnecessary_unwrap \
+-A clippy::unnecessary_mut_passed \
+-A clippy::unnecessary_cast \
+-A clippy::unused_unit \
+-A clippy::unusual_byte_groupings \
+-A clippy::useless_asref \
+-A clippy::useless_conversion \
+-A clippy::vec_init_then_push \
+-A clippy::while_let_on_iterator \
+-A clippy::writeln_empty_string \
+-A clippy::wrong_self_convention
```

Comment:
fix EOL please

Reply:
Yeah the problem with that is that we can't tell `rust-analyzer` which lints to annoy about and which not, right?